app/move/steps/create.js

```diff
@@ -16,7 +16,7 @@ const {
 const personSearchStep = {
   controller: PersonSearch,
   buttonText: 'actions::search',
-  template: 'person-search',
+  template: '../person-search',
```

Comment:
Don't think there is a reason to create custom templates for these steps. It can be updated via the field/step configuration.

Reply:
I haven't created templates -- I have _moved_ them. They are shared between `create` and `assign` but incongruously, they were in the `create` template folder.